src/MicrosoftTeams.ts

```diff
@@ -2119,4 +2119,43 @@ namespace microsoftTeams {
       ]);
     }
   }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about all members in a chat
+   */
+  export interface ChatMembersInformation {
+    members: ChatMember[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about a chat member
+   */
+  export interface ChatMember {
+    /**
+     * The user name of the user principal.
+     */
+    upn: string;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * ------
+   * Allows an app to retrieve information of all chat members
```

Comment:
This value can be spoofed by a malicious attacker, right? I think it needs the same dire warnings as with the identity hints in tab context: apps should not make security decisions based on this value without considering the ramifications of the spoofing threat.

Reply:
Added warning